src/main/java/com/oltpbenchmark/api/Procedure.java

```diff
@@ -120,8 +120,6 @@ else if (is != null) {
         else {
             pStmt = conn.prepareStatement(stmt.getSQL());
         }
-
```

Comment:
Please try avoiding these changes. These changes create unnecessary traces in git blame